src/internal/client/client.go

```diff
@@ -98,6 +102,7 @@ type APIClient struct {
 	AdminAPIClient
 	TransactionAPIClient
 	DebugClient
+	LogsClient
```

Comment:
I know that you’re just copying the existing pattern here, which is fine, but I think that you should consider just adding a member rather than embedding here. I don’t think that there’s a need for embedding in this case — it just means being able to call `logs.APIClient` methods on a `Client` instance, when they don’t overlap with some other embedded type’s methods.

Reply:
no client changes now